src/main/java/macrobase/conf/MacroBaseDefaults.java

```diff
@@ -70,4 +70,5 @@
     public static final List<String> CONTEXTUAL_DISCRETE_ATTRIBUTES = new ArrayList<String>();
     public static final List<String> CONTEXTUAL_DOUBLE_ATTRIBUTES = new ArrayList<String>();
     public static final Double OUTLIER_STATIC_THRESHOLD = 3.0;
+    public static final Integer NUM_MIXTURES = 1;
```

Comment:
I was just curious if the default is 1 why you wouldn't use the BayesianNormalDensity estimator instead? This one fits using EM but...?